scripts/common_startup.sh

```diff
@@ -197,10 +197,8 @@ if [ $SKIP_CLIENT_BUILD -eq 0 ]; then
         if [ "$GIT_BRANCH" = "0" ]; then
             SKIP_CLIENT_BUILD=1
         else
-            # Compare hash.
-            githash=$(git rev-parse HEAD)
-            statichash=$(cat static/client_build_hash.txt)
-            if [ "$githash" = "$statichash" ]; then
+            # Check if anything has changed in client/ since the last build
+            if git diff --quiet $(cat static/client_build_hash.txt).. -- client/; then
```

Comment:
If we drop the two `..` in the diff range specification, it includes the working directory changes, otherwise I think this is comparing against the head commit.

Reply:
Thanks @dannon, rebased with the change.